EntAppProject/src/main/resources/templates/JBCreateJobListing.html

```diff
@@ -11,20 +11,20 @@
 </head>
 
 <body>
- 
+
 <h1><span>Job</span>Book</h1>
 
 <h2>Create A Job Listing</h2>
 
 <form action="/JBHomescreen">
 Description of Job:
 <br>
-<input style=position:left; type="text" id="userBio">
-<br> 
+<input style="float:left;" type="text" id="userBio">

```

Comment:
Style attributes need quotes, and also, `position:left` does not exist in CSS. I switched it to `float:left`, but unsure if that is what you want.